dev/io.openliberty.security.jakartasec.4.0.internal/src/io/openliberty/security/jakartasec40/identitystore/InMemoryIdentityStoreDefinitionWrapper.java

```diff
@@ -0,0 +1,421 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2025 IBM Corporation and others.
```

Comment:
```suggestion
 * Copyright (c) 2025 IBM Corporation and others.
```
Is this coming from another classes that it should include 2017?

Reply:
The class structure comes from `DatabaseIdentityStoreDefinitionWrapper.java` class within package `com.ibm.ws.security.javaeesec`, but it has been changed fairly significantly. 

I copied the header as was:

"Copyright (c) 2017, 2022 ..."

and updated 2022 to 2025.

Should I leave it, do as you suggest or add: "Copyright (c) 2017, 2022, 2025 ..."?

Reply:
It is always one date or two dates.  Never 3.  It is first date that the code was written, to the last date it was changed.  I am guessing it should just be `2025`.